src/test/java/uk/gov/hmcts/ccd/domain/service/createcase/SubmitCaseTransactionTest.java

```diff
@@ -82,19 +88,19 @@ void setup() {
         MockitoAnnotations.initMocks(this);
 
         submitCaseTransaction = new SubmitCaseTransaction(caseDetailsRepository,
-                                                          caseAuditEventRepository,
-                                                          caseTypeService,
-                                                          callbackInvoker,
-                                                          uidService,
-                                                          securityClassificationService,
-                                                          caseUserRepository);
+                                                            caseAuditEventRepository,
+                                                            caseTypeService,
```

Comment:
Can you check that spaces are intended

Reply:
Applied formatting using IDE via CCD.xml formatting file